bundles/org.openhab.binding.enocean/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -8,4 +8,20 @@
 	<description>The EnOcean binding integrates the EnOcean system into openHAB.</description>
 	<connection>local</connection>
 
+	<discovery-methods>
+		<discovery-method>
+			<service-type>usb</service-type>
+			<match-properties>
+				<match-property>
+					<name>remote</name>
+					<regex>false</regex>
+				</match-property>
```

Comment:
Just for my understanding, what does this criterai mean exactly ?

Reply:
There are two types of USB serial 'stick' devices -- namely those which are physically connected locally to the PC, and those which are connected remotely from another PC via RFC2217 'socat'. Some bindings (such as ZigBee and Z-Wave) can use both. But I am not sure if this binding EnOcean can use the latter, which is why we currently exclude it from the suggestion finder. However if @fruggy83 can confirm that it also works via remote, then we can remove this filter.

Reply:
Ok, understood.